File: contests/final-brasileira-2024/a.cpp
// Append and Panic!
// Este problema foi passado por nossa equipe na Final Brasileira. Entretanto, utilizamos linguagem de Programação Python.

#include <bits/stdc++.h>
using namespace std;

int main(){
    string l = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    map<char, int> dic;
    for(int i=0; i<26; i++)
        dic[l[i]] = i;
    int o[26] = {0};
    string s;
    cin>>s;
    for(unsigned int i=0; i<s.size(); i++)
        o[dic[s[i]]] += 1;
    int a=0;
    for(int i=0; i<26; i++)
        a += max(o[i] - 1, 0);
    cout<<a<<endl;
    return 0;
}

File: contests/final-brasileira-2024/a.py
# Append and Panic!
# Este foi o exato código utilizado para passar o problema na Final Brasileira

s = input()
let = {}
for l in s:
    try:
        let[l] += 1
    except KeyError:
        let[l] = 1
tam = 0
for v in let.values():
    tam += v - 1
print(tam)